src/api/clearlyDefined.js

```diff
@@ -9,10 +9,13 @@ export const API_DEVELOP = 'https://dev-api.clearlydefined.io'
 export const API_PROD = 'https://api.clearlydefined.io'
 export const apiHome = process.env.REACT_APP_SERVER || getServiceDefaultUrl()
 
+// Uncomment the next instruction to set your current enviroment as local, thus using API_LOCAL
+// DON'T COMMIT THIS FILE IF YOU CHANGED THIS INSTRUCTION
```

Comment:
We've done several tests trying to set process.env.NODE_ENV properly when starting the application, without success.
Given that, the options for a dev to switch enviroment are:
- set process.env.NODE_ENV in this file
- set API_LOCAL under the 'development' value of the switch.

Neither of them are the best way, but documenting the process maybe would be enough?

Reply:
I highly recommend the first one `set process.env.NODE_ENV` rather than changing back and forth the switch, @storrisi @jeffmcaffer .

Besides that, local configuration should generally be in .gitignore and not committed every time (other than the first time, where those act as a template).

CI/CD processes should populate the configuration accordingly (via e.g. SECRETS, etc).

Not sure which approach we are currently using on ClearlyDefined, but in this specific instance I think it might be beneficial to set process.env.NODE_ENV in the file.

Actually, an even better option, given we are using `create-react-app`, might be to create a .env file so that it will become possible to set process.env.NODE_ENV the right way (see https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-development-environment-variables-in-env)

Reply:
The problem with process.env in CRA is that it has to be set at BUILD time, not run time. Since the CRA setup creates a static website there is code in there that does substitution etc. for NODE_ENV. I like the .env file approach. Again here the key point is that it is clear to the user 

* that there are multiple possible configurations
* how to get the different configs setup

Reply:
Opened #257 to  address this